pkg/client/cli/cmd_gatherlogs.go

```diff
@@ -105,10 +108,18 @@ func (gl *gatherLogsArgs) gatherLogs(cmd *cobra.Command, ctx context.Context, st
 		daemonLogs = append(daemonLogs, "daemon")
 	case "user":
 		daemonLogs = append(daemonLogs, "connector")
-	case "False":
+	case "None":
 	default:
-		return errcat.User.New("Options for --daemons are: all, root, user, or False")
+		return errcat.User.New("Options for --daemons are: all, root, user, or None")
 	}
+	// Add metadata about the request so we can track usage + see which
+	// types of logs people are requesting more frequently.
+	// This also gives us an idea about how much usage this command is
+	// getting.
+	scout.SetMetadatum("daemon_logs", daemonLogs)
+	scout.SetMetadatum("traffic_manager_logs", gl.trafficManager)
+	scout.SetMetadatum("traffic_agent_logs", gl.trafficAgents)
+	scout.Report(log.WithDiscardingLogger(ctx), "Used gather-logs")
```

Comment:
pls rename this metric so that it's `snake_cased` and also add it to `TELEMETRY.md` at the root of the repo